### PR TITLE
Np 46412 change country and keyword model

### PR DESCRIPTION
**Pull request by torarnet**
Changed datamodel for keyword and country to a model that is more in tune with our other data models for these types of data. Need to coordinate with frontend as this will break existing keyword query. The other endpoints, country list, keyword fetch and keyword creation are not used yet so should not be a problem

### CODE REVIEW

File: cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/Keyword.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    @JsonCreator
+    public Keyword(@JsonProperty(CONTEXT) String context,
+                   @JsonProperty(TYPE) String type,
+                   @JsonProperty(ID) URI id,
+                   @JsonProperty(IDENTIFIER) String identifier,
+                   @JsonProperty(LABELS) Map<String, String> labels) {
+
+        super(context, type, id, identifier, labels);
+    }
```
Comment (brinxmat) [inline cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/Keyword.java:30]: Do we end up with a situation where we can create Keywords with a different type?

File: cristin-keyword/src/test/java/no/unit/nva/cristin/keyword/query/queryKeywords.feature [hunk 2]
```diff
@@ -46,12 +46,14 @@ Feature: API tests for keywords query
     When method GET
     Then status 200
     And match response == '#object'
-    And match response['@context'] == '#present'
+    And match response['@context'] == 'https://bibsysdev.github.io/src/keyword-search-context.json'
```
Comment (brinxmat) [inline cristin-keyword/src/test/java/no/unit/nva/cristin/keyword/query/queryKeywords.feature:49]: Do you have such a context?

File: cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/Keyword.java (hunk 1 above)
Comment (torarnet) [reply cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/Keyword.java:30]: No, not now at least

File: cristin-keyword/src/test/java/no/unit/nva/cristin/keyword/query/queryKeywords.feature (hunk 2 above)
Comment (torarnet) [reply cristin-keyword/src/test/java/no/unit/nva/cristin/keyword/query/queryKeywords.feature:49]: No, can you help me with that one day?

**Review by anetteOlli — APPROVED** [review]

**Review by StigNorland — APPROVED** [review]

**Pull request merged**